src/js/edu-benefits/1990e/components/IntroductionPage.jsx

```diff
@@ -16,22 +16,47 @@ class IntroductionPage extends React.Component {
     return (
       <div className="schemaform-intro">
         <FormTitle title="Manage Your Education Benefits"/>
```

Comment:
Good catch--changed to "Apply for Education Benefits"

I'm going to go ahead and merge this and let Alex reply later on whether or not to make the telephone numbers links. Molly resolved the other questions I had.